### PR TITLE
docs: fix typos, make code style more consistent

### PR DESCRIPTION
**Pull request by tomdaly**
## What does this change?

Module: React
Week(s): 1, cheatsheet

## Description

- Fix typos
- Make code style consistent

## Who needs to know about this?

@CodeYourFuture/react

## Rendered Pages

-----
[View rendered docs/guides/react-cheatsheet.md](https://github.com/CodeYourFuture/syllabus/blob/react/module-typo-link-fixes/docs/guides/react-cheatsheet.md)
[View rendered docs/react/week-1/homework.md](https://github.com/CodeYourFuture/syllabus/blob/react/module-typo-link-fixes/docs/react/week-1/homework.md)
[View rendered docs/react/week-1/mentors.md](https://github.com/CodeYourFuture/syllabus/blob/react/module-typo-link-fixes/docs/react/week-1/mentors.md)

### CODE REVIEW

**Comment by 40thieves** [comment]
Thanks! Maybe someone from @CodeYourFuture/react could review 🙂

File: docs/guides/react-cheatsheet.md [hunk 1]
```diff
@@ -42,28 +50,27 @@ As before! You can set up automatic Netlify deployment from GitHub in the same w
...
-| ------------------------------------ | ------------- |
-| &quot;publish directory&quot;        | build/        |
-| &quot;build command&quot;            | npm run build |
+| **Important Netlify build settings** | **Value**       |
+| ------------------------------------ | --------------- |
+| "publish directory"                  | `build/`        |
+| "build command"                      | `npm run build` |
 
-Note that deployment will be a bit slower than with simpler projects as it has to &quot;build&quot; your React project.
+Note that deployment will be a bit slower than with simpler projects as it has to "build" your React project.
```
Comment (DomVinyard) [inline docs/guides/react-cheatsheet.md]:
```suggestion
Note that your deployment doesn't happen instantly. Netlify needs some time to "build" your React project. The build process turns your beautiful human-readable React Components into optimised machine-readable code that can run in a web browser. 
```

File: docs/guides/react-cheatsheet.md [hunk 2]
```diff
@@ -42,28 +50,27 @@ As before! You can set up automatic Netlify deployment from GitHub in the same w
...
 
 ### Error when launching app with npm start
 
-- Are you in the right directory?
-  Or, this could be a problem with node modules
-- For example, you may see an error with a message saying &quot;module not found&quot;
-- Try to npm install the missing package
-- If this fails, delete your node_modules folder and package-lock.json and re-run npm install|
+- Are you in the right directory? Run `ls`, you should see a "package.json" listed
+- If you see an error with a message saying "module not found", try to npm install the missing package
```
Comment (DomVinyard) [inline docs/guides/react-cheatsheet.md]:
```suggestion
- If you see the error message "module not found", run `npm install module_name` to install the missing package.
```

**Review by DomVinyard — DISMISSED** [review]
lgtm, left a couple of suggestions, not sure if helpful 🙂

**Review by ehwus — DISMISSED** [review]
Sorry Tom I only just saw this, everything looks sensible to me!

**Comment by 40thieves** [comment]
I think this could be merged now @tomdaly. I think you should have permission, so I'll let you do the honours 🙂

**Comment by gregdyke** [comment]
I fixed a merge conflict at the url for netlify deployment instructions has changed

**Comment by tomdaly** [comment]
@ehwus @40thieves I think the master merge dismissed approving reviews, would you mind reviewing again?

**Review by SallyMcGrath — APPROVED** [review]
LGTM

**Pull request merged**